js/feed.js

```diff
@@ -1,141 +1,213 @@
-(function () {
-  const intervals = [
-    { label: "year", seconds: 31536000 },
-    { label: "month", seconds: 2592000 },
-    { label: "day", seconds: 86400 },
-    { label: "hour", seconds: 3600 },
-    { label: "minute", seconds: 60 },
-    { label: "second", seconds: 0 }
-  ];
+pageToLoad = 1;
 
-  function timeSince(date) {
-    const seconds = Math.floor((Date.now() - date.getTime()) / 1000);
-    const interval = intervals.find(i => i.seconds < seconds);
-    const count = Math.floor(seconds / interval.seconds);
-    return count + " " + interval.label + (count !== 1 ? "s" : "") + " ago";
-  }
+function createActivityFeed(amountOfPosts, loadMoreButton) {
+    var intervals = [
+      { label: "year", seconds: 31536000 },
+      { label: "month", seconds: 2592000 },
+      { label: "day", seconds: 86400 },
+      { label: "hour", seconds: 3600 },
+      { label: "minute", seconds: 60 },
+      { label: "second", seconds: 0 }
+    ];
```

Comment:
Here I had to add a check 
```js
  if (!window.__githubApiPage) {
    window.__githubApiPage = 1;
  }
```
because the variable would get reset to 1 every time the function was called. Is this ok?